content/docs/user-guide/basic-concepts/workspace.md

```diff
@@ -2,7 +2,47 @@
 name: Workspace
 match: [workspace]
 tooltip: >-
-  Directory containing all your project files e.g. raw datasets, source code, ML
-  models, etc. Typically, it's also a Git repository. It will contain your DVC
+  The directory containing all your project files, e.g., the raw data, source
+  code, ML models. Typically, it's also a Git repository. It contains your DVC
   project.
 ---
+
+# Workspace
+
+A data science project can consist of data obtained from many distinct sources.
+These may be split into multiple files or directories or (as the project
+structure needs) have different versions for different requirements, e.g., a
+smaller / simplified version might be required in prototyping for faster
+feedback and shorter training times. A single workspace to manage all artifacts
+of a project is desirable, although versioning needs and managing dependencies
+make it increasingly difficult.
+
+DVC allows a single directory to contain all your project artifacts. The
+workspace is the directory containing the _visible_ part of your
+<abbr>project</abbr>, e.g., the raw data, source code, model files. You can have
+multiple versions of data, models, and other kinds of artifacts within the
+workspace and limit your focus to a subset of these. You can record your
+progress in a commit and analyze your data and model history. DVC provides a
+_machine learning file system_ to manipulate your data and models using its
```

Comment:
This "ML file system" term is pretty tricky. No need to force it (just skip it if you can't find a correct way to use it).

I can only think of something like "DVC turns your project into a sort of machine learning file system for..." but not sure.